.travis.yml

```diff
@@ -1,15 +1,29 @@
 language: cpp
 compiler:
   - gcc
-python:
-  - "2.7"
+jobs:
```

Comment:
I don't know much about travis, but the following [does not work](https://github.com/mmisono/ubpf/runs/474778656) (use python2 in both cases).
```
python:
  - 2.7
  - 3.5
```
I guess this is because language is cpp. So here I manually install python.